kratos/includes/model_part_io.h

```diff
@@ -268,7 +264,7 @@ class KRATOS_API(KRATOS_CORE) ModelPartIO : public IO
 
     ///@}
 
-protected:
+private:
```

Comment:
This is supposed to be private

Reply:
Are you compiling the Metis application? If I remember correctly, some of the methods in this section were used there...

If they are not needed, please disregard. I agree that it would be better to make the next section private.

Reply:
I will try